recipes/miniaudio/all/conanfile.py

```diff
@@ -1,19 +1,19 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.layout import basic_layout
 import os
-import textwrap
-
-required_conan_version = ">=1.43.0"
 
+required_conan_version = ">=1.53.0"
 
 class MiniaudioConan(ConanFile):
     name = "miniaudio"
     description = "A single file audio playback and capture library."
-    topics = ("miniaudio", "header-only", "sound")
-    homepage = "https://github.com/mackron/miniaudio"
-    url = "https://github.com/conan-io/conan-center-index"
     license = ["Unlicense", "MIT-0"]
-    settings = "os", "compiler", "build_type", "arch"
-
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mackron/miniaudio"
+    topics = ("miniaudio", "header-only", "sound")
```

Comment:
No need for the library name in the topic :)

```suggestion
    topics = ("audio" "header-only", "sound")
```